docs/full-node/run-a-full-terra-node/build-terra-core.mdx

```diff
@@ -10,6 +10,18 @@ Terra Core is the official Golang reference implementation of the Terra node sof
 
 ## Get the Terra Core source code
 
+<Admonition type="tip" icon="💡" title="Core versions and heights">
+
+If you are syncing a node from genesis you will need to use the appropriate core version for the block height of the respective chain you are syncing.
+
+| Network       | Type    | Core Version at Genesis | Block Height where core version changes        | Core version to use after fork height                                                                             |
```

Comment:
@Jared-TFL @terran6 is "fork height" the proper term here? 

Disregard if it is.

Reply:
@evanorti  @Jared-TFL Ya, I might personally use Network Name, Network Type, Core Version at Genesis, Block Height of Core Version Update, Updated Core Version.

Reply:
It is most accurately a fork height.
So, I suppose

| Network       | Type    | Core Version at Genesis | Fork height        | Core version after fork height